platform/security/expansion/security-expansion-commands/pom.xml

```diff
@@ -50,11 +50,11 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
+                        <Karaf-Commands>*</Karaf-Commands>
```

Comment:
The commands are not registered or available without this. I think there is another way of doing this as well with another build plugin but this seemed simpler.